server/documents/modules/toast.html.eco

```diff
@@ -873,7 +923,11 @@ themes      : ['Default']
             <td>mini</td>
             <td>Can hold a string to be added to the image class. <code>mini</code>, <code>tiny</code>, <code>small</code> and <code>avatar</code> are supported out of the box</td>
         </tr>
-
+        <tr>
+            <td>context</td>
+            <td>body</td>
+            <td>Selector or jquery object specifying the area to to attach the toast container to</td>
```

Comment:
Typo error with double `to` "Selector or jquery object specifying the area to ~~to~~ attach the toast container to".

Reply:
Fixed by [`3343aae` (#259)](https://github.com/fomantic/Fomantic-UI-Docs/pull/259/commits/3343aae3a74238f183e1767684038b9180dc7224) 🙂